### PR TITLE
Hz 0205 chester

### PR DESCRIPTION
**Pull request by CHesterHHH**

### CODE REVIEW

**Review by Timguau — APPROVED** [review]

File: src/main/java/com/eta/houzezbackend/auth/JwtAuthenticationEntryPoint.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.eta.houzezbackend.auth;
```
Comment (yuanningd) [inline src/main/java/com/eta/houzezbackend/auth/JwtAuthenticationEntryPoint.java:1]: I guess we can delete this

File: src/main/java/com/eta/houzezbackend/controller/AgentController.java [hunk 2]
```diff
@@ -23,14 +23,20 @@ public AgentGetDto signUp(@Valid @RequestBody AgentSignUpDto agentSignUpDto) {
         return agentService.signUpNewAgent(agentSignUpDto);
     }
 
+    @PostMapping("/sign-in")
+    @ResponseStatus(HttpStatus.CREATED)
```
Comment (yuanningd) [inline src/main/java/com/eta/houzezbackend/controller/AgentController.java:27]: created?

File: src/test/java/com/eta/houzezbackend/controller/AgentControllerTests.java [hunk 3]
```diff
@@ -6,14 +6,16 @@
...
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.http.MediaType;
 
 
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-class AgentControllerTests extends ApplicationIntTest{
+@AutoConfigureMockMvc(addFilters = false)
```
Comment (yuanningd) [inline src/test/java/com/eta/houzezbackend/controller/AgentControllerTests.java:17]: Move this annotation to` ApplicationIntTest`

File: src/test/java/com/eta/houzezbackend/AuthTest.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+class AuthTest extends ApplicationIntTest {
```
Comment (yuanningd) [inline src/test/java/com/eta/houzezbackend/AuthTest.java:26]: then this one does not extend `ApplicationIntTest`

File: src/main/java/com/eta/houzezbackend/auth/JwtAuthenticationEntryPoint.java (hunk 1 above)
Comment (CHesterHHH) [reply src/main/java/com/eta/houzezbackend/auth/JwtAuthenticationEntryPoint.java:1]: ok

File: src/main/java/com/eta/houzezbackend/controller/AgentController.java (hunk 2 above)
Comment (CHesterHHH) [reply src/main/java/com/eta/houzezbackend/controller/AgentController.java:27]: change to 200OK

File: src/test/java/com/eta/houzezbackend/controller/AgentControllerTests.java (hunk 3 above)
Comment (CHesterHHH) [reply src/test/java/com/eta/houzezbackend/controller/AgentControllerTests.java:17]: done

**Pull request merged**